### PR TITLE
Update vgis.yml

### PR DESCRIPTION
**Pull request by bdorry**
# Description

What does this pull request achieve? Why do we need it?

# Checklist

- [ ] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/vonage-business-cloud/vgis.yml [hunk 1]
```diff
@@ -682,7 +677,7 @@ paths:
                 - OUTBOUND
         - in: query
           name: states
-          description: Filter by state. (Mix SMS/CALL states)
+          description: 'Return calls from this index in the response (Default: ACTIVE)'
```
Comment (mheap) [inline definitions/vonage-business-cloud/vgis.yml]: Should this be `with this state`?

File: definitions/vonage-business-cloud/vgis.yml [hunk 2]
```diff
@@ -693,67 +688,62 @@ paths:
...
                 - REMOTE_HELD
-                - DETACHED
-                - REJECTED
-                - CANCELLED
-                - ANSWERED
-                - MISSED
         - in: query
           name: offset
-          description: Page number, from beginning
+          description: Return calls from this index in the response
```
Comment (mheap) [inline definitions/vonage-business-cloud/vgis.yml]: Is this multiplied by `size` or is it an absolute offset?

File: definitions/vonage-business-cloud/vgis.yml [hunk 3]
```diff
@@ -693,67 +688,62 @@ paths:
...
+          description: 'Sort in either ascending or descending order (Default: ASC)'
           schema:
             type: string
             enum:
               - DESC
               - ASC
         - in: query
           name: sort
-          description: Sort property
+          description: 'Sort calls by property'
```
Comment (mheap) [inline definitions/vonage-business-cloud/vgis.yml]: Should this be `the field to sort results by`?

File: definitions/vonage-business-cloud/vgis.yml (hunk 1 above)
Comment (bdorry) [reply definitions/vonage-business-cloud/vgis.yml]: Looks like copy paste got me

File: definitions/vonage-business-cloud/vgis.yml (hunk 2 above)
Comment (bdorry) [reply definitions/vonage-business-cloud/vgis.yml]: Updated

File: definitions/vonage-business-cloud/vgis.yml (hunk 3 above)
Comment (bdorry) [reply definitions/vonage-business-cloud/vgis.yml]: Updated

**Pull request merged**